generator/src/main/java/org/apache/kafka/message/MessageDataGenerator.java

```diff
@@ -482,7 +486,9 @@ private String readFieldFromReadable(FieldType type) {
             return "readable.readUUID()";
         } else if (type.isString()) {
             return "readable.readNullableString()";
-        } else if (type.isBytes()) {
+        } else if (type.isBytes() && useByteBuffer) {
```

Comment:
Ditto here.

Reply:
Sorry, but I don't know what "Ditto" means :)

I assume, that you want `!useByteArray` be removed.
If yes, then fixed.